config/canonical.json

```
@@ -8686,6 +8686,8 @@
     "tags": {
       "amenity": "fuel",
       "brand": "Cenex",
+      "brand:wikidata": "Q5011381",
+      "brand:wikipedia": "https://en.wikipedia.org/wiki/CHS_Inc.",
```

Thanks for looking this up for us!  Can you format the tag like this:
```suggestion
      "brand:wikipedia": "en:CHS Inc.",
```